content/pl/logs.md

```diff
@@ -0,0 +1,16 @@
+## XI. Logi
+### Traktuj logi jako strumień zdarzeń
+
+*Logi* zapewniają wgląd w zachowanie działającej aplikacji. W środowiskach korzystających z serwera zazwyczaj są zapisywane na dysku (plik "logfile"); jednak jest to tylko wybrany format zapisu.
+
+Logi są [strumieniem](http://adam.heroku.com/past/2011/4/1/logs_are_streams_not_files/) gromadzących się, uporządkowanych w czasie zdarzeń zebranych ze strumieni wyjściowych wszystkich uruchomionych procesów i usług wspierających. Logi w swojej pierwotnej formie występują zazwyczaj w formacie tekstowym z jedną linią na zdarzenie (wyjątkiem jest jednak [backtrace](https://en.wikipedia.org/wiki/Stack_trace), który może zajmować wiele lini). Logi nie mają określonego początku ani końca, napływają nieustannie podczas działania aplikacji.
+
+**Aplikacja 12factor nie odpowiada za przekierowywanie i zapis swojego strumienia wyjściowego.** Nie powinna też zapisywać czy zarządzać plikami logów. Zamiast tego, każdy działające proces zapisuje swój strumień zdarzeń, niebuforowany, do `stdout`. Podczas pracy w lokalnym środowisku developer może przeglądać strumień w tle, w oknie swojego terminala, by obserwować zachowanie aplikacji.
+
+We wdrożeniu stagingowym czy produkcyjnym, każdy strumień procesów zostanie przechwycony przez środowisku wykonawcze, dołączony do pozostałych i skierowany do jednego lub wielu miejsc, w celu przeglądania i długoterminowego zapisu. Miejsca zapisu nie są widoczne ani konfigurowalne przez aplikację. W całości zarządza nimi środowisko wykonawcze. W tym celu można skorzystać z dostępnych open-source routerów logów (takich jak [Logplex](https://github.com/heroku/logplex) lub [Fluent](https://github.com/fluent/fluentd)).
```

Comment:
1. `s/środowisku/środowisko`
2. Nie powinno być przecinka po `wielu miejsc`
3. `s/konfigurowalne/konfigurowane` brzmi lepiej
4. `przez aplikację. W całości` -> `przez aplikację - w całości` 
5. `z dostępnych open-source routerów logów`? Wysil się proszę bardziej, to brzmi strasznie :)